app/screens/WalletNavigator/screens/CreateWallet/PinConfirmation.tsx

```diff
@@ -42,7 +45,11 @@ export function PinConfirmation ({ route }: Props): JSX.Element {
     setTimeout(() => {
       MnemonicEncrypted.toData(copy.words, copy.network, copy.pin)
         .then(async encrypted => {
+          await MnemonicWords.encrypt(words, pin)
           await setWallet(encrypted)
+
+          // temp disabled biometric, forward data to biometric enrolment page after more test
+          // navigation.navigate('EnrollBiometric', { pin, encrypted, words })
```

Comment:
It will remain unused.
No changes required on TransactionAuthorization, it behave as it is when no biometric enrolled.

I think we are confident this code works find and merge-able, but just need more thorough testing for edge cases and different devices.